content/applications/websites/ecommerce/getting_started/catalog.rst

```diff
@@ -1,37 +1,94 @@
+
+.. _ecommerce/started/catalog/catalog:
 
 Product Catalog
 ===============
 
-All your published items show up in your catalog page (or *Shop* page).
+When opening your **shop** for the first time, all items are published one after the other. You need
+to customize it so that you display the information you need.
 
-.. image:: catalog/shop.png
+.. image:: catalog/shop-overview.png
    :align: center
+   :alt: Overview of the catalog page without any customization nor navigation options.
 
-Most options are available in the *Customize* menu: display attributes,
-website categories, etc.
+You can display more information or change the page layout with the :guilabel:`Customize`
+menu. To access it, go to :menuselection:`Edit --> Customize`.
 
-.. image:: catalog/shop_customize.png
-   :align: center
+Once there, many customization options appears:
 
-Highlight a product
-===================
+- :guilabel:`Layout`: Allows to choose how products are displayed. It includes the number of
+  products on the   page and per line, the style, and the size of your product pictures.
+- :guilabel:`Prod. Desc.`: Allows to display the products' description if there's one.
+- :guilabel:`Left Panel`: Allows to add more navigation options, such as accessing specific product
+  categories or attributes (colors, sizes, etc.).
+- :guilabel:`Top Bar`: Allows to add more display options for customers, such as how product are
+  sorted by, or the layout of the page (grid, list, etc.).
+- :guilabel:`Default Sort`: Allows to choose how products are sorted when opening the shop.
+- :guilabel:`Buttons`: Allows to add quick actions, such as add to cart, add to favorite, etc.
+
+Let's see some of those in action.
+
+.. _ecommerce/started/catalog/layout:
+
+Page layout
+===========
+
+Under the :guilabel:`Layout` section, you can change several display options. For this example,
+let's change the number of products per page from 20 to 21, and the number of products per line
+from 4 to 3 with the :guilabel:`Size` option.
+
+For the :guilabel:`Style` and :guilabel:`Images Size`, let's choose :guilabel:`Cards` and
+:guilabel:`Landscape (4/3)` to add borders to our pictures and reduce the size of each block.
+
+.. _ecommerce/started/catalog/panel:
 
-Boost the visibility of your star/promoted products: push them to top, make them
-bigger, add a ribbon that you can edit (Sale, New, etc.). Open the Shop page, switch
-to Edit mode and click any item to start customizing the grid.
+Left Panel
+==========
 
-See how to do it: https://www.odoo.com/openerp_website/static/src/video/e-commerce/editing.mp4
+To give your customers more navigation options, you can use the :guilabel:`Left Panel` options.
+Let's display :guilabel:`Categories` and enable :guilabel:`Collapse Categories` to add a navigation
+menu for customers.
+
+Since giving the more information about your products is important, let's also enable the
```

Comment:
```suggestion
Since giving more information about your products is important, let's also enable the
```
I think you left a "the" when rewriting the sentence